### PR TITLE
[FOR REVIEW ONLY] made single-server app

### PR DESCRIPTION
**Pull request by pranavpandey1998official**
this is review PR for single server do not merge this 
defaults sever is https://ilarion.rocket.chat please change that from app.json for review

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
   "name": "RocketChatRN",
-  "displayName": "RocketChatRN"
+  "displayName": "RocketChatRN",
+  "server": "https://ilarion.rocket.chat"
```
Comment (diegolmello) [inline app.json]: Change to open.

Comment (pranavpandey1998official) [reply app.json]: test will start failing failling I cannot do anything

Comment (diegolmello) [reply app.json]: You mean e2e? Don't worry.

Comment (pranavpandey1998official) [reply app.json]: done

**Comment by diegolmello** [comment]
Merged on `single-server`

**Pull request closed without merging**